test/cljam/test_common.clj

```diff
@@ -189,8 +198,14 @@
 (def test-vcf-no-samples-file "test-resources/vcf/test-no-samples.vcf")
 (def test-vcf-complex-file "test-resources/vcf/test-v4_3-complex.vcf")
```

Comment:
Create new resources and add them to the Git repository.
```shell
cat test-resources/vcf/test-v4_3-complex.vcf | bgzip -c -l9 > test-resources/vcf/test-v4_3-complex.vcf.gz
tabix -p vcf test-resources/vcf/test-v4_3-complex.vcf.gz
git add test-resources/vcf/test-v4_3-complex.vcf.gz*
```

Then add tests with these files.